libraries/Microsoft.Bot.Builder.Azure/CosmosDbStorage.cs

```diff
@@ -159,26 +127,26 @@ public async Task DeleteAsync(string[] keys, CancellationToken cancellationToken
         {
             if (keys == null || keys.Length == 0)
```

Comment:
do you really need this test and throw?
what would be wrong with no keys returning no records, that would be what i'd expect.